### PR TITLE
feat: Add accounts endpoints

### PR DESCRIPTION
**Pull request by sridharvoruganti**
Related to [#1999](https://github.com/mojaloop/project/issues/1999)

-  implement  `GET /accounts/{id}`
-  implement  `PUT  /accounts/{id}`
-  implement  `PUT /accounts/{id}/error` endpoints

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
     "@mojaloop/central-services-health": "10.6.0",
     "@mojaloop/central-services-logger": "10.6.0",
     "@mojaloop/central-services-metrics": "9.5.0",
-    "@mojaloop/central-services-shared": "11.4.3",
+    "@mojaloop/central-services-shared": "v11.5.6",
```
Comment (kleyow) [inline package.json:110]: nit: v

File: src/domain/accounts.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      headers,
+      sourceDfspId,
+      destinationDfspId,
+      method,
+      payload,
+      Enum.Http.ResponseTypes.JSON,
+      childSpan
+    )
+
+    Logger.info(`authorizations::forwardAccountsIdRequest - Forwarded accounts request : ${userId} from ${sourceDfspId} to ${destinationDfspId}`)
```
Comment (kleyow) [inline src/domain/accounts.ts:156]: nit: accounts

**Review by kleyow — DISMISSED** [review]
hmm...lgtm

File: package.json (hunk 1 above)
Comment (sridharvoruganti) [reply package.json]: Ok done

File: src/domain/accounts.ts (hunk 2 above)
Comment (sridharvoruganti) [reply src/domain/accounts.ts]: fixed

File: src/domain/accounts.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+ * @throws {FSPIOPError} Will throw an error if no endpoint to forward the accounts requests is
+ *  found, if there are network errors or if there is a bad response
+ * @returns {Promise<void>}
+ */
+export async function forwardAccountsIdRequestError (
+  path: string,
+  headers: HapiUtil.Dictionary<string>,
+  userId: string,
+  error: APIErrorObject,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (eoln) [inline src/domain/accounts.ts:66]: instead of any use unknown

File: src/domain/accounts.ts [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+import { Util as HapiUtil } from '@hapi/hapi'
+import Logger from '@mojaloop/central-services-logger'
+
+import {
+  Enum,
+  Util,
+  FspEndpointTypesEnum,
+  RestMethodsEnum
+} from '@mojaloop/central-services-shared'
+// eslint is complaining about these imports. not sure why.
```
Comment (eoln) [inline src/domain/accounts.ts:35]: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md

looks like the linter is unable to resolve `~/shared/config` path,but  tsconfig.js setup is ok

Which linter is complaining, the vs code one plugin, or `npm run lint` ?

For me, this line could fix the problem: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/.eslintrc.js#L40

there is no such line in thidparty-api-adapter: https://github.com/mojaloop/thirdparty-api-adapter/blob/master/.eslintrc.js

**Review by eoln — CHANGES_REQUESTED** [review]

File: src/domain/accounts.ts (hunk 4 above)
Comment (sridharvoruganti) [reply src/domain/accounts.ts]: Ok fixed now

File: src/domain/accounts.ts (hunk 3 above)
Comment (sridharvoruganti) [reply src/domain/accounts.ts:62]: If we change span type to unknown,  lint complains at span?.getChild, then also need to typecast it to any

**Review by eoln — APPROVED** [review]

**Pull request merged**